services/gossip/service.go

```diff
@@ -74,15 +76,15 @@ func (s *service) SendLeanHelixPrePrepare(input *gossiptopics.LeanHelixPrePrepar
 		RecipientMode: gossipmessages.RECIPIENT_LIST_MODE_BROADCAST,
 		Topic:         gossipmessages.HEADER_TOPIC_LEAN_HELIX,
 		LeanHelix:     gossipmessages.LEAN_HELIX_PRE_PREPARE,
-		NumPayloads:   1,
+		NumPayloads:   uint32(len(input.Message.BlockPair.TransactionsBlock.SignedTransactions)),
```

Comment:
I think there's a bug here that was here before. Num of payloads should be `len(txs) + 1`, as the block header is the first payload, no?